src/internal/pfsdb/pfsdb.go

```diff
@@ -14,85 +14,133 @@ import (
 )
 
 const (
-	reposPrefix       = "/repos"
-	commitsPrefix     = "/commits"
-	branchesPrefix    = "/branches"
-	openCommitsPrefix = "/openCommits"
-	mergesPrefix      = "/merges"
-	shardsPrefix      = "/shards"
+	reposCollectionName       = "repos"
+	branchesCollectionName    = "branches"
+	commitsCollectionName     = "commits"
+	openCommitsCollectionName = "open_commits"
 )
 
+var ReposTypeIndex = &col.Index{
+	Name: "type",
+	Extract: func(val proto.Message) string {
+		return val.(*pfs.RepoInfo).Repo.Type
+	},
+}
+
+var ReposNameIndex = &col.Index{
+	Name: "name",
```

Comment:
Do either of these names work?  Aren't they both reserved words?

Reply:
The index name is mapped by the `PostgresCollection` to the field `idx_name`